images/runtime/dotnetcore/7.0/base.buster.Dockerfile

```diff
@@ -34,6 +34,8 @@ ENV ASPNETCORE_URLS=http://+:80 \
     # Enable detection of running in a container
     DOTNET_RUNNING_IN_CONTAINER=true \
     PATH="/opt/dotnetcore-tools:${PATH}"
+#Bake in client certificate path into image to avoid downloading it
+ENV PATH_CA_CERTIFICATE="/etc/ssl/certs/ca-certificate.crt"
```

Comment:
Maybe I missed this in a previous PR or email thread or something, but how are these environment variables actually used? I don't see anywhere that we are using them, and I'm also not sure why we don't supply it in all of our runtime images (e.g. php-fpm and ruby don't have this, other dotnet base files don't have it, etc.)

Reply:
This request is from the App Services team to create an environment variable for client certs so they are baked into runtime images for Python 3.10, PHP 8.1, Node 18 and .Net 7 Preview 7

Please see email thread - Bundling Azure MySQL/Postgres client cert in App Service Oryx Images for more context.

Reply:
Hmm looking at that email thread, I'm not finding `PATH_CA_CERTIFICATE`. Did we send that to our partners to use? So they would be using it, and we don't need to do anything but give them the path of the certificate?

Reply:
Looks like they want env for PATH_CA_CERTIFICATE baked into images Antares would release with ANT:99

Antares provided the path to the certificate file. Essentially, the ask on this work item was to just add an env variable for easy reference.